recipes_source/inference_tuning_on_aws_graviton.rst

```diff
@@ -0,0 +1,360 @@
+            nn.Linear(4096, 4096),
+            nn.ReLU(),
+            nn.Linear(4096, 11008),
+            nn.ReLU(),
+            nn.Linear(11008, 10),
+        )
+
+    def forward(self, x):
+        x = self.flatten(x)
+        logits = self.linear_relu_stack(x)
+        return logits
+
+4. Let's create an instance of ``MyNeuralNetwork``, and move it to the device:
+
+.. code-block:: python
+
+    model = MyNeuralNetwork().to(device)
+    print(model)
+
+Next, let's get the prediction probabilities by passing them through an instance of the ``nn.Softmax`` module:
+
+.. code-block:: python
+
+    X = torch.rand(1, 64, 64, device=device)
+    logits = model(X)
+    pred_probab = nn.Softmax(dim=1)(logits)
+    y_pred = pred_probab.argmax(1)
+    print(f"Predicted class: {y_pred}")
+
+output:
+
+.. code-block::
+
+    Predicted class: tensor([2])
+
+Our network functionality is verified. Next, we will profile the performance. Lets' check two different scenarios: small and large batch dimensions.
+
+**Scenario 1:** A larger batch dimension, for example 256:
+
+.. code-block:: python
+
+    # warm it up first and loop over multiple times to have enough execution time
+
+    X = torch.rand(256, 64, 64, device=device)
+
+    with torch.set_grad_enabled(False):
+        for _ in range(50):
+            model(X) #Warmup
+        with profile(activities=[ProfilerActivity.CPU]) as prof:
+            with record_function("mymodel_inference"):
+                for _ in range(100):
+                    model(X)
+
+    print(prof.key_averages().table(sort_by="self_cpu_time_total"))
+
+
+Following is the profiler output with the default PyTorch configuration:
+
+.. table:: profiler output with the torch default configuration
+   
+======================  ============   ===========  =============  ===========  ============  ============  
+                  Name    Self CPU %      Self CPU   CPU total %     CPU total  CPU time avg    # of Calls  
+======================  ============   ===========  =============  ===========  ============  ============
+           aten::addmm        97.61%       15.813s        98.61%       15.977s      53.255ms           300  
+       aten::clamp_min         1.09%     177.032ms         1.09%     177.032ms     885.160us           200  
+           aten::copy_         1.00%     162.054ms         1.00%     162.054ms     540.180us           300  
+     mymodel_inference         0.22%      35.738ms       100.00%       16.201s       16.201s             1  
+          aten::linear         0.02%       2.955ms        98.66%       15.985s      53.282ms           300  
+               aten::t         0.01%       2.421ms         0.03%       5.043ms      16.810us           300  
+            aten::relu         0.01%       2.356ms         1.11%     179.388ms     896.940us           200  
+======================  ============   ===========  =============  ===========  ============  ============
```

Comment:
Can you indent lines 117- 127 under the table directive for correct recognition. More info [here](https://docutils.sourceforge.io/docs/ref/rst/directives.html#table)

Same comment for all table directives in the PR.